File: src/main/java/br/com/springboot/model/Evento.java
package br.com.springboot.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;
import java.util.Date;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name = "evento")
public class Evento {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    private String nome;

    private String descricao;

    private Date data;

    private String local;

    private String participantes;

    private Enum Tipo;




}

File: C#/APIAgenda/Models/Evento.cs
namespace APIAgenda.Models
{
    public class Evento
    {
        private string _nome;
        private string _descricao;
        private string _data;
        private string _local;
        private string partipantes;
    }
    
        public string Nome
        {
            get => _nome;
            set => _nome = value?.Trim();
        }

        public double Data { get; set; }
        
        public double Local { get; set; }
        
        public double Participantes { get; set; }
        
        public double Descricao { get; set; }
    }
}